### DISCUSSION TITLE
Plot pole figures of host grain and twin separately

### DISCUSSION THREAD

**Discussion by hannahcunning** [mtex-toolbox/mtex#2227] [Ask Anything]
Hello, 

I have been trying to plot the orientations of the host grains and the twins separately (like figure below), but since the twinning is defined as grain boundaries (I have been following the documentation on twinning boundaries) I cannot figure out how to extract the ebsd orientations.  

Help is appreciated, thank you. 

![Screenshot 2024-09-19 152412](https://github.com/user-attachments/assets/895096a7-13b9-4f9e-91e4-2c862689f5f7)

**Comment by ralfHielscher** [comment]
Maybe this docu page https://mtex-toolbox.github.io/GrainMerge.html does help you. It shows one way of differentiating between host and twin. What was your idea?

Ralf.

**Comment by hannahcunning** [comment]
The results of the twin and host process in the Grain Merge documentation gives you a grainBoundary file, which means that I cannot automatically differentiate which side of the grain boundary is the twin and which is the host. What I want is a grain2d file of all of the host grains (H in image) and another grain2d file of all of the twins (T in image) so that I can plot pole figures of only the host grains/only the twins. My data is some heavily twinned calcite and I don't want to manually select the 800+ identified twins from the merge function. The "checkInside" function does not work because the twins intersect the host grain boundaries.  
![Screenshot 2024-09-20 121005](https://github.com/user-attachments/assets/eec298f6-0c95-476b-a2c2-844f799d7562)
